src/adagrams.js

```diff
@@ -1,15 +1,143 @@
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  const letterPool = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+    }
+  let hand = [];
+  let randomLetter = "";
+  while (hand.length < 10) {
+    randomLetter = getRandomLetter(letterPool);
+    if (letterPool[randomLetter] > 0)  {
+      letterPool[randomLetter] -= 1;
+            hand.push(randomLetter.toUpperCase());
+    }
+  } 
+  return hand;
 };
```

Comment:
One other suggestion here is that you are currently weighing all 26 letters the same, but that is not an accurate spread of how the letters shake out! We are more likely to grab an e than an x, but this doesn't currently represent that. What we would need to do is make a list that holds each individual letter and then pull from there!